ERCS/eip-locakable1155.md

```diff
@@ -0,0 +1,158 @@
+---
+eip:
```

Comment:
```suggestion
eip: 7721
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.

Reply:
thanks for eip number, updated!